plugin/evm/vm_test.go

```diff
@@ -32,15 +32,16 @@ import (
 	"github.com/ava-labs/avalanchego/snow"
 	"github.com/ava-labs/avalanchego/snow/choices"
 	"github.com/ava-labs/avalanchego/snow/consensus/snowman"
-	"github.com/ava-labs/avalanchego/utils/constants"
 	"github.com/ava-labs/avalanchego/utils/formatting"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/version"
 	"github.com/ava-labs/avalanchego/vms/components/chain"
 
 	engCommon "github.com/ava-labs/avalanchego/snow/engine/common"
+	avaGoConstants "github.com/ava-labs/avalanchego/utils/constants"
```

Comment:
nit: can we use `engConstants` for consistency?

Reply:
done

Reply:
lol I had the exact opposite nit

Reply:
uh... after looking at the package names I think avaConstants make more sense. since engCommon = "engine/common"